pkg/errors/errors.go

```diff
@@ -47,6 +47,7 @@ var (
 	Unknown              error = errors.New("Unknown Reason")
 	DeleteFailed         error = errors.New("Delete Failed")
 	DeletePending        error = errors.New("Delete Pending")
+	WMIError             error = errors.New("WMI Error")
 )
 
 func GetErrorCode(err error) string {
```

Comment:
For all the errors defined in the WMI module, we should probably do something like the following in GetErrorCode():

if IsNotFound(err) **|| wmi.IsNotFound(err)** {
    return "NotFound"
}

Reply:
And so on...